.github/workflows/run_tests.yml

```
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Check code style
```

У Вас имена action'ов одинаковые. Это вызывает неоднозначность.
```suggestion
name: Check tests
```